src/components/Personrad.tsx

```diff
@@ -104,6 +105,9 @@ export const Personrad = (props: PersonradProps): ReactElement => {
       </Column>
       <Column xs={'2'}>{veilederName}</Column>
       <Column xs={'1'}>{oppfolgingstilfelleLengthInWeeks}</Column>
+      <Column xs={'1'}>
+        <FristColumn personData={personData} />
+      </Column>
```

Comment:
Hadde det vært greiere å hatt `showAvventerFrist` her? Da blir det lettere å se om denne kolonnen faktisk renderes eller ikke i parenten.
Går litt på [dette](https://react.dev/learn/conditional-rendering#conditionally-returning-nothing-with-null:~:text=In%20practice%2C%20returning%20null%20from%20a%20component%20isn%E2%80%99t%20common%20because%20it%20might%20surprise%20a%20developer%20trying%20to%20render%20it.%20More%20often%2C%20you%20would%20conditionally%20include%20or%20exclude%20the%20component%20in%20the%20parent%20component%E2%80%99s%20JSX.%20Here%E2%80%99s%20how%20to%20do%20that!) som @JMLindseth har vært inne på tidligere som jeg synes hørtes ut som en god idé 😊 Hva tenker du?